configure

```diff
@@ -1905,9 +1905,9 @@ public:
     	return false;
 #else
     	QString str =
+    		"#include<X11/extensions/scrnsaver.h>\\n"
```

Comment:
This is a generated file. It doesn't make sense to change it manually.

Reply:
If this point of view, we can remove the file ;)